mediabridge/download_data.py

```diff
@@ -0,0 +1,67 @@
+import os
+import tarfile
+
+import wget
+
+"""
+For our scripts to work, we need to download the Netflix prize data. As this file is ~500 MB and the licensing terms are dubious, we decided early on not to include it as part of the repo. It is however available on the Internet Archive here: https://archive.org/details/nf_prize_dataset.tar
```

Comment:
I think this comment only needs the first paragraph. The rest is a description of how to write this very script.